src/lib/response-triggers/finish.ts

```diff
@@ -54,20 +56,25 @@ export default class Finish extends Trigger {
 			});
 		__logger.debug('[Finish.createMessage] Got winners and responses');
 
+		let message = `## Winners\n\n` + winnersMarkdown + '\n\n';
+		message += resultsMarkdown;
+
+		this.spark.messages.create({
+			markdown: message,
+			roomId: room
+		});
+
 		try {
 			await this.pmResultsService.pmResults();
 		} catch(error) {
 			__logger.error(`[Finish.createMessage] Error PMing results: ${error.message}`);
 			return { markdown: `Error while trying to PM results` };
 		}
 
-		var reset = await this.reset.createMessage();
+		//var reset = await this.reset.createMessage();
 
-		let message = `## Winners\n\n` + winnersMarkdown + '\n\n';
-		message += resultsMarkdown;
-		message += '\n\n' + reset.markdown;
 		return {
-			markdown: message
+			markdown: 'PMs successfully sent.'
```

Comment:
Don't think this is necessary. Just return nothing.

Reply:
To be more clear, I think adding the extra message in here is a bad design. We want to avoid sending status messages like this when not called for.

Reply:
i like having it for now so that we know it all went well. seeing as the other messages are now sent before this we have no other confirmation that it didnt crash somewhere in the middle of the pms

Reply:
If it crashes during the PMs it'll throw an error and come out with an error response